### DISCUSSION TITLE
How to stop new plots from being added to queue?

### DISCUSSION THREAD

**Discussion by Kaveman42** [swar/Swar-Chia-Plot-Manager#358] [Q&A] [answered]
Is it possible to stop the plotter from adding new plots to the queue and only finish the plots currently in progress? I'd like to update my config file but would hate to stop the progress of all the current plots in the queue.

**Comment by raid5** [comment, marked as answer]
Just update your config and then run `python manager.py restart` to pick up the changes. Or if you actually want to stop the manager from creating new plots, you can run `python manager.py stop`

**Reply by Kaveman42** [reply]
Thank you, I used the restart command and it worked. If I use the stop command does it continue the items in queue or halt those as well?

**Reply by cdeniz** [reply]
> Thank you, I used the restart command and it worked. If I use the stop command does it continue the items in queue or halt those as well?

How did you update the config file ?

**Reply by Kaveman42** [reply]
In the Swar Plot Manager folder there is a file called config.yaml. You can edit that file with the settings you need.

**Reply by cdeniz** [reply]
actualy my question what is the setting?

**Reply by farkeytron** [reply]
Another tip... run the "manager.py view" in a separate CMD window... and run the "manager.py stop/start/restart" commands in another window. This way, the "view" window will continue showing the progress of the plotting, even if you stop the manager.
